app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -201,12 +199,14 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
   return (
     <>
       <PageHeading>{t('in_person_proofing.headings.location')}</PageHeading>
-      {arcgisSearchEnabled && (
+      <div aria-live="polite">
         <AddressSearch onAddressFound={handleFoundAddress} registerField={registerField} />
```

Comment:
this is really a thought for design - but wondering if the search field should clear the user's input after clicking the search button. And if the "no matches found" text has enough prominence on the page. Right a user see's the address they input 2x on the page and it's hard to notice the "no match" text.

Reply:
oh wait, is the 2nd address shown on the page the one that was actually used by the search function?/ the closest match to the input? that's unclear. I'll add these thoughts in slack so design can comment.

Reply:
crossing out the portions that are likely not applicable since the 2nd address is for debugging